css/main.css

```diff
@@ -0,0 +1,127 @@
+html,
+body {
+    margin: 0;
+    padding: 0;
+    font-family: calibri;
+}
+
+a {
+    text-decoration: none;
+    color: #c0392b;
+}
+
+a:hover {
+    color: #e74c3c;
+}
+
```

Comment:
Not a big issue your css got lots of white space

Reply:
my visual studio code has this pep8 kind of linter that does that to the code